gadopt/diagnostics.py

```diff
@@ -36,6 +38,7 @@ class GeodynamicalDiagnostics:
       T_min: Minimum temperature in domain
       T_max: Maximum temperature in domain
       ux_max: Maximum velocity (first component, optionally over a given boundary)
+      uk_min: Minimum velocity (vertical component, optionally over a given boundary)
```

Comment:
Is `uk` a reasonable choice here? I am wondering if it is obvious to everyone that it represents the vertical component. Could `uv` be better? Or should we be more explicit in the naming (e.g. `u_vert_min`)?

Reply:
renamed to `uv_min` I think this is all going to get looked at again in #299 so I think this ok for now.